arkouda/numeric.py

```diff
@@ -2201,6 +2203,51 @@ def count_nonzero(pda):
     elif pda.dtype == str:
         return sum((pda != "").astype(np.int64))
 
+def array_equal(pda_a: pdarray, pda_b: pdarray, equal_nan: bool = False):
+    """
+    Compares two pdarrays for equality.
+    If neither array has any nan elements, then if all elements are pairwise equal,
+    it returns True.
+    If equal_Nan is False, then any nan element in either array gives a False return.
+    If equal_Nan is True, then pairwise-corresponding nans are considered equal.
+
+    Parameters
+    ----------
+    pda_a : pdarray
+    pda_b : pdarray
+    equal_nan : boolean to determine how to handle nans, default False
+
+    Returns
+    -------
+    boolean
+      With string data: False if one array is of type ak.str_ and the other isn't,
+                        True if both are ak.str_ and they match.
+      With numeric data: True if neither array has any nan elements, and all elements pairwise equal.
+                         False if equal_Nan is False, and either array has any nan element.
+                         True if equal_Nan is True, all non-nan elements are pairwise equal,
+                                            and all nans in pda_a correspond to nans in pda_b
+
+    Examples
+    --------
+    >>> a = ak.randint(0,10,10,dtype=ak.float64)
+    >>> b = a
+    >>> ak.array_equal(a,b)
+    True
+    >>> b[9] = np.nan
+    >>> ak.array_equal(a,b)
+    False
+    >>> a[9] = np.nan
+    >>> array_equal(a,b)
```

Comment:
Should be `ak.array_equal(a,b)` (instead of `array_equal(a,b)`) to match the others.